test/plugin/scenarios/mongodb-3.x-scenario/support-version.list

```diff
@@ -19,3 +19,15 @@
 3.4.2
 3.4.3
 3.5.0
+3.7.0
```

Comment:
Just a context, what happens in 3.5.0-3.7.0? Is the code different? Like an anonymous class used?

Reply:
3.6.x using anonymous class to implements OperationExecutor, it is difficult to intercept it.

Reply:
Good to doc here. I just want official confirmation

Reply:
Not in source code, just in the PR should be good. Don't need to update.